plugins/basic/menu.py

```diff
@@ -0,0 +1,39 @@
+import inspect
+import os
+import sys
+
```

Comment:
would be great with a docstring here explaining what this file is and when it is being executed.